### PR TITLE
Introduce graph-metrics-exporter Cronjob

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/metrics-exporter/issues/704

## Does this require new deployment ?

- [x] Deployment for Test and Stage `AICoE/aicoe-cd` and Prod `operate-first/argocd-apps`.

## Description
Add new component graph-metrics-exporter CronJob.

Tested in test environmnet already and metrics are correctly sent to pushgateway

### CODE REVIEW

**Comment by pacospace** [comment]
We can even schedule it every 15 minutes I think!

**Comment by pacospace** [comment]
Depends-On: https://github.com/thoth-station/storages/pull/2368

File: graph-metrics-exporter/base/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  name: graph-metrics-exporter
+  labels:
+    app: thoth
+    component: graph-metrics-exporter
+spec:
+  schedule: "@hourly"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```
Comment (harshad16) [inline graph-metrics-exporter/base/cronjob.yaml:14]: lets reduce these to more smaller value
```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 1
```

**Review by harshad16 — COMMENTED** [review]
/lgtm
just one minor comment.

**Comment by pacospace** [comment]
@harshad16 PTAL 💯

**Review by harshad16 — APPROVED** [review]
/lgtm :+1:

**Pull request merged**